javascript/javascript1/week1/goodboy-oldboy/goodboy-oldboy.js

```diff
@@ -0,0 +1,13 @@
+const dogYearOfBirth = 2012;
+const dogYearFuture = 2045;
+const dogYear = dogYearFuture - dogYearOfBirth;
+const shouldShowResultInDogYears = false
+
+if (shouldShowResultInDogYears === true) {
```

Comment:
No need to update it, but this can also be written as 
`if(shouldShowResultInDogYears)`